setup/freebsd/resources/jenkins

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# $FreeBSD: $
+#
+# Based on the jenkins init script
+#
+# PROVIDE: jenkins-iojs
+# REQUIRE: LOGIN
+# KEYWORD: shutdown
+
+#
+# Configuration settings for jenkins in /etc/rc.conf:
+#
+# jenkins_enable (bool):
+#   Set to "NO" by default.
+#   Set it to "YES" to enable jenkins
+#
+
+. /etc/rc.subr
+
+name="jenkins"
+rcvar=jenkins_enable
+
+load_rc_config "${name}"
+
+
+OSTYPE="freebsd"
+NODE_COMMON_PIPE="/home/iojs/test.pipe"
+LOCALHOST="{{interface}}"
+
+jenkins_jnlpurl="https://jenkins-iojs.nodesource.com/computer/iojs-voxer-freebsd101-{{id}}/slave-agent.jnlp"
+jenkins_secret="{{secret}}"
+jenkins_jar="/home/iojs/slave.jar"
+jenkins_log_file="/home/iojs/jenkins_console.log"
+jenkins_args="-jnlpUrl ${jenkins_jnlpurl} -secret ${jenkins_secret}"
+jenkins_user="iojs"
+jenkins_group="iojs"
+
+pidfile="/var/run/jenkins/jenkins.pid"
```

Comment:
It's good practise to own the pidfile with the same user as the daemon. In this case it's not needed but I tried to stay close to [freebsd's own init script](https://svnweb.freebsd.org/ports/head/devel/jenkins/files/jenkins.in?revision=368098&view=markup).